src/web/lib/braze/checkBrazeDependencies.test.ts

```diff
@@ -161,6 +161,39 @@ describe('checkBrazeDependecies', () => {
 		}
 	});
 
+	it('fails when the user is not signed in', async () => {
+		setWindow({
+			guardian: {
+				config: {
+					switches: {
+						brazeSwitch: true,
+					},
+					page: {
+						brazeApiKey: 'fake-api-key',
+						isPaidContent: false,
+					},
+				},
+			},
+		});
+		mockBrazeUuid = null;
+		mockConsentsPromise = Promise.resolve(true);
```

Comment:
Are these globals? I don't see them being used. Not anything for this PR but perhaps a seed worth planting around the api for that function - could these values be passed in?

Reply:
Hey great question. These are defined at the top of the file and are used by `jest.mock` as we're mocking at the module level. I think you're right, it's probably a sign that the interface to this function isn't quite right. In the short term (this PR) in this test we don't care about the `mockBrazeUuid` so I'll remove that line.